ecosystem/typescript/sdk_v2/src/core/account.ts

```diff
@@ -77,7 +77,7 @@ export class Account {
    * Derives an account with random private key and address
    *
    * @param args.scheme SigningScheme - type of SigningScheme to use.
-   * Currently only Ed25519, MultiEd25519, and Secp256k1 are supported
+   * Currently only Ed25519, and Secp256k1 are supported
```

Comment:
```suggestion
   * Currently Ed25519, and Secp256k1 are supported
```

nit but remove `only` so people don't think the SDK is incomplete or something

Reply:
Wait why did you remove `MultiEd25519` here? It should be here I think